config/locales/en.yml

```diff
@@ -197,6 +211,11 @@ en:
     change_email:
       subject: "[%{app_name}] Email changed"
       part1: "%{login}, the email associated with your account has been changed to %{email}"
+    reset_password:
```

Comment:
I don't think this belongs in this PR... it seems to be the old version of the password reset email, which we changed when we moved to Devise.